### PR TITLE
fix segment,callin,http doc incorrect description

### PR DESCRIPTION
**Pull request by leewenchen**

### CODE REVIEW

File: docs/http.md [hunk 1]
```diff
@@ -9,37 +9,45 @@
 ## 属性
 | 属性 | 类型 | 是否必须 | 备注 |
 |--------|--------|--------|--------|
-|   url   | [expr](datatype.md)  | √ |  发起网络请求的目标地址，可以是http或https开头 |
-|   media   | [expr](datatype.md)  | x  |  post数据的媒体类型，默认为`text/plain`,可参考[MIME类型](https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Basics_of_HTTP/MIME_types) |
-|   post   | [expr](datatype.md)  | x |  |
+|   url   | [string](datatype.md)  | √ |  发起网络请求的目标地址，可以是http或https开头 |
```
Comment (shenruisi) [inline docs/http.md]: This type should be expr

File: docs/http.md [hunk 2]
```diff
@@ -9,37 +9,45 @@
...
+|   media   | [string](datatype.md)  | x  |  post数据的媒体类型，默认为`text/plain`,可参考[MIME类型](https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Basics_of_HTTP/MIME_types) |
```
Comment (shenruisi) [inline docs/http.md]: This type should be expr

File: docs/http.md [hunk 3]
```diff
@@ -9,37 +9,45 @@
...
 ### post请求
 ```yaml
-- http: {url: '"http://127.0.0.1/echo"', post: '"ping"' path: "/resp"}
+- http: {url: '"https://www.baidu.com"', path: '/resp', post: '"plain text body"'}
+- echo: '$/resp'
 ```
 
 ### 定义media的post请求
 ```yaml
+- http: {url: '"http://127.0.0.1/formPost"', media: '"application/x-www-form-urlencoded"', post: '"content=hello"', path: '/resp'}
```
Comment (shenruisi) [inline docs/http.md]: Try to get a reachable url to give the examples.
Maybe like that

```yaml
- http: {url: '"https://reqbin.com/echo/post/form"', media: '"application/x-www-form-urlencoded"', post: '"content=hello"', path: '/resp'}
- echo: '$/resp'
```

**Review by shenruisi — CHANGES_REQUESTED** [review]
Try to update with suggestions.

File: docs/callin.md [hunk 4]
```diff
@@ -2,16 +2,16 @@
...
 ```
 您可以通过**callin**标签来调用定义好的segment函数
 
 ## 属性
 | 属性 | 类型 | 是否必须 | 备注 |
 |--------|--------|--------|--------|
-|   seg   | [expr](datatype.md)  | √ |   |
-|   0...49   | [expr](datatype.md)  | x  |   |
-|   args   | [expr](datatype.md)  | x | |
+|   package   | [string](datatype.md)  | √ |   |
```
Comment (shenruisi) [inline docs/callin.md]: This type should be expr

File: docs/callin.md [hunk 5]
```diff
@@ -2,16 +2,16 @@
...
+|   seg   | [string](datatype.md)  | √ |   |
```
Comment (shenruisi) [inline docs/callin.md]: This type should be expr

File: docs/callin.md [hunk 6]
```diff
@@ -2,16 +2,16 @@
...
+|   0...49   | [string](datatype.md) | x  |   |
```
Comment (shenruisi) [inline docs/callin.md]: This type should be expr

**Review by shenruisi — CHANGES_REQUESTED** [review]
Try to update with suggestions.

File: docs/segment.md [hunk 7]
```diff
@@ -24,9 +24,9 @@
 ```yaml
 - segment:
     package: '"trico"'
-    name: '"_concat"' 
+    name: '"concat"' 
     args: {0: '/x',1: '/y'} 
     body:
       - put: {value: '$/x+$/y',path: '/z'}
       - return: '$/z'
```
Comment (turbe) [inline docs/segment.md]: adjust indent, preferred 4 spaces

Comment (leewenchen) [reply docs/segment.md]: got it

**Pull request merged**